src/api/java/baritone/api/Settings.java

```diff
@@ -653,13 +653,6 @@ public final class Settings {
      */
     public final Setting<Boolean> logAsToast = new Setting<>(false);
 
-    /**
-     * The time of how long the message in the pop-up will display
-     * <p>
-     * If below 1000L (1sec), it's better to disable this
-     */
-    public final Setting<Long> toastTimer = new Setting<>(5000L);
```

Comment:
Not needed/possible anymore?

Reply:
probably possible but the timer is encoded as a final field in `SystemToastId`

so we'd have to revert back to implementing a custom `Toast` subclass or do some other hack

reusing system toasts is much simpler and i don't think its worth the maintenance burden doing the alternatives

Reply:
Ok, fine.